src/kirby/components/list/directives/bg-color-type.directive.ts

```diff
@@ -0,0 +1,54 @@
+import { Directive, HostBinding, Input } from '@angular/core';
```

Comment:
Is there a reason for creating this directive and not use using the `themeColor` directive? It looks like a 100% duplicate of it.

Are we expecting to both have `themeColor` and `bgColorType` on some elements?

If we have to have two directives, lets have a common base class as we don't want to maintain to list of our colors.

Reply:
The difference is that themeColor is styling both the background and the text. We might be able to just use one and apply different styling, as they are only setting a css class.

Reply:
I ended up using just themeColor directive